source/Applications/Advanced/HandEyeCalibration/README.md

```diff
@@ -0,0 +1,34 @@
+# Hand Eye Calibration
+
+To fully understand Hand-Eye Calibration, please see the [tutorial][Tutorial-url] in our Knowledge Base.
+
+-----------------
+
+[HandEyeCalibration][HandEyeCalibration-url]:
+
+* Application which walks through the collection of calibration poses
+   1. Provide robot pose to application (manual entry)
+   2. Application takes an image of the calibration object, and calculates pose
+   3. Move robot to new position, enter command to Add Pose
+   4. Repeat i.-iii. until 10-20 pose pairs are collected
+   5. Enter command to perform calibration and return a **Transformation Matrix**
+
+[ZividHandEyeCalibration][ZividHandEyeCalibration-url]
```

Comment:
I agree. We have the same in C++, but it is a bit strange to have this link here (since its not a sample). @SatjaSivcev, do you know why it was added here in the first place? I think we either should remove it or expose the code as a proper sample.

Reply:
@torbsorb was the one who added it. It was written as a tutorial so I think it should stay there. Perhaps we should clarify that it comes with the installer for Win and is part of tools deb?

Reply:
> @torbsorb was the one who added it. It was written as a tutorial so I think it should stay there. Perhaps we should clarify that it comes with the installer for Win and is part of tools deb?

Good idea

Reply:
Included a sentence there I say that this sample comes with the installer in Windows and is part of tools deb in Ubuntu